ee/orbit/pkg/securehw/securehw_linux.go

```diff
@@ -262,93 +260,37 @@ func (t *tpm2TEE) selectBestECCCurve() (tpm2.TPMECCCurve, string) {
 	return tpm2.TPMECCNistP384, "P-384"
 }
 
-// writeBlobsToFiles writes the TPM public and private blobs to the specified file paths
-func (t *tpm2TEE) writeBlobsToFiles(publicBlob tpm2.TPM2BPublic, privateBlob tpm2.TPM2BPrivate) error {
-	t.logger.Debug().
-		Str("public_path", t.publicBlobPath).
-		Str("private_path", t.privateBlobPath).
-		Msg("writing TPM blobs to files")
-
-	// Marshal the public blob
-	publicData := tpm2.Marshal(publicBlob)
-	if err := os.WriteFile(t.publicBlobPath, publicData, 0o600); err != nil {
-		return fmt.Errorf("write public blob to %s: %w", t.publicBlobPath, err)
+func (t *tpm2TEE) saveTPMKeyFile(privateKey tpm2.TPM2BPrivate, publicKey tpm2.TPM2BPublic) error {
+	k := keyfile.NewTPMKey(
+		keyfile.OIDOldLoadableKey,
```

Comment:
You probably want `keyfile.OIDLoadableKey` here. I realize now that the `README.md` example is using the wrong OID.